yorkie/rpc/server.go

```diff
@@ -125,3 +136,26 @@ func (s *Server) listenAndServeGRPC() error {
 
 	return nil
 }
+
+// Validate validates port number and checks if files exists
+func (c *Config) Validate() error {
+	var err error
+	if c.Port < 0 || 65535 < c.Port {
+		return fmt.Errorf("%w: %d", ErrWrongPort, c.Port)
+	}
```

Comment:
```suggestion
	if c.Port < 1 || 65535 < c.Port {
		return fmt.Errorf("must be between 1 and 65535, given %d : %w", c.Port, ErrInvalidPortNumber)
	}
```

1. A port of 0 is not a normal value.
2. It would be nice to place the sentinel error at the end: https://blog.golang.org/go1.13-errors#TOC_5